src/index.js

```diff
@@ -27,38 +26,37 @@ class Bootstrap extends EventEmitter {
     this._timer = null
   }
 
-  start (callback) {
+  start () {
```

Comment:
I know that there were no docs prior. However, could you add those here? It would be really helpful and should be easy to add

Reply:
Sure no problem